Formula/unar.rb

```diff
@@ -17,6 +17,16 @@ class Unar < Formula
 
   depends_on xcode: :build
 
+  uses_from_macos "llvm" => [:build, :test]
+
```

Comment:
Based on linkage test for Linux, looks like might want `uses_from_macos "bzip2"`.